libs/blocks/figure/figure.css

```diff
@@ -69,6 +69,7 @@
   .figure .figure {
     margin: 0 auto;
     max-width: 800px;
+    width: fit-content;
```

Comment:
![image](https://github.com/user-attachments/assets/d96e2e24-e405-420c-8066-314a2c28ed39)

But i believe this can be changed to just figure tag inside a figure class.